templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -5,58 +5,54 @@
  */
 #}
 {% spaceless %}
```

Comment:
Let's take the chance to replace these with `{% apply spaceless %}` as this one used above is deprecated already.